static-analysis/teammates-stylelint.yml

```diff
@@ -383,7 +383,8 @@ rules:
     # Selector list
 
     # Require a newline or disallow whitespace after the commas of selector lists
-    # selector-list-comma-newline-after
+    # One selector per line.
```

Comment:
Now that this comment is here under another comment, it might be better to provide more context e.g. `Used to enforce one selector per line.`